packages/form/addon/components/cf-field/input/number-separator.hbs

```diff
@@ -0,0 +1,13 @@
+<input
+  type="text"
+  class="uk-input
+    {{if @field.isInvalid 'uk-form-danger'}}
+    {{if this.disabled 'uk-disabled'}}"
+  name={{@field.pk}}
+  id={{@field.pk}}
+  value={{this.displayValue}}
+  placeholder={{@field.question.raw.placeholder}}
+  readonly={{this.disabled}}
+  pattern={{this.pattern}}
```

Comment:
There is no `this.pattern` in the component. Should it be an argument?

Reply:
Leftover from something I tried. Removed it.